Makefile

```diff
@@ -2,6 +2,9 @@ BIN_OUTPUT_PATH = bin/$(shell uname -s)-$(shell uname -m)
 
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 
+NDK_ROOT = ~/android-ndk-r25c
```

Comment:
So a few concerns here. 

1. This doesn't match the r26 version used in CI. Should it?
2. How/where do people get this? It should probably be added to setup or, probably better (as most people won't need it) a new target for setup-android or something.
3. Paths like this should be absolute (for system installed things) or paths within the repo, not arbitrary locations in a user's home folder, as that forces them to clutter things up. I'd suggest `etc/android-ndk-r26` or similar (and adding that to git exclude.)
4. Should probably use ?= so it's easy to override with ENV (both for developers and in CI.)

Reply:
oops -- we're targeting a fairly old compat target in the actual build (android30, but it's a different versioning scheme than the 25/26) so both work, but updated to match

I think there's no recommended install location for droid ndk -- normally the android studio gui installs in a subdirectory

this is a big versioned dep, so it felt more idiomatic to put it in a central location, but will move it down if you want

Reply:
`?=` changed in 646adbac

Reply:
for 'where to get it' -- can we hold off until someone tries to build it? it will depend on linux vs mac and may take some research on the mac side

Reply:
I think it's fine to say it has to build in canon (just like the linux binaries do) so you can make it linux-host only for now if that simplifies things. I'd assume it'd just be the same zip file as CI.

Reply:
moved to etc + added download command in 80cdc13a